semgrep/semgrep/formatter/text.py

```diff
@@ -197,6 +252,52 @@ def _build_text_timing_output(
             rule_id = truncate(rule_id, col_lim) + ":"
             yield f"{with_color('yellow', f'{rule_id:<71}')} run time {total_time:.4f}  match time {match_time:.4f}"
 
+        # Output other file information
+        ANALYZED = "Analyzed:"
+        FAILED = "Failed:"
+        headings = [ANALYZED, FAILED]
+        max_heading_len = max(len(h) for h in headings) + 1  # for the space
+
+        def add_heading(heading: str, lines: List[str]) -> List[str]:
```

Comment:
I think there might be some native methods that could make alignment easier if we know the max group size: https://stackoverflow.com/questions/8234445/format-output-string-right-alignment

Reply:
Leaving it because I like only having to change the headings